### DISCUSSION TITLE
Prevent Browser From Closing After Script Ends

### DISCUSSION THREAD

**Discussion by FBT-MBK** [seleniumbase/SeleniumBase#2418] [Q&A] [answered]
I'm Using SeleniumBase Driver module I want my browser to keep running after my script ends, Is it possible ?
I want Multiple browsers at the same time without closing previous one.
Thanks in Advance.

**Comment by mdmintz** [comment, marked as answer]
`breakpoint()` for pausing your script at the end (which keeps the browser open).

`pytest --rs` for reusing the browser session between tests in the same thread.
